src/utils/wallet.js

```diff
@@ -131,6 +131,7 @@ const wallet = {
     try {
       walletUtils.wordsValid(words);
     } catch(e) {
+      console.error(e);
```

Comment:
Do we still need this console?
If so, maybe we should move it to the closure on 138/139, to avoid presenting the full error in case of `InvalidWords`.

Reply:
We don't need this one, but I left it to make debugging easier.
Although having invalid words is expected in one case (importing a seed from user input), I think we should still log this error.